bluetooth/src/androidLibMain/kotlin/UUID.kt

```diff
@@ -0,0 +1,29 @@
+package com.splendo.kaluga.bluetooth
+
+import android.os.ParcelUuid
+
+actual data class UUID(val uuid: ParcelUuid) : BaseUUID {
+
+    override val uuidString: String
+        get() = uuid.toString()
+
+    override val isValid: Boolean
+        get() = true
+
+    override fun equals(other: Any?): Boolean {
+        if (this === other) return true
+        if (javaClass != other?.javaClass) return false
+
+        other as UUID
```

Comment:
This looks strange to me

Reply:
Pretty sure this is the IDE generated equals function